test/SIL.XForge.Scripture.Tests/Services/SFProjectServiceTests.cs

```diff
@@ -340,6 +340,28 @@ public async Task ReserveLinkSharingKeyAsync_GenerateNewKeyIfReserved()
         );
     }
 
+    [Test]
+    public void GetProjectSecret_ReturnsSecret()
+    {
+        var env = new TestEnvironment();
+        var shareKey = "abcd";
```

Comment:
This should be a constant in `SFProjectServiceTests`, like:
```csharp
private const string ShareKey = "abcd";
```
(and any other copy-pasted constant string you want to clean up in this class?)